app/assets/javascripts/views/query_view.js.coffee

```diff
@@ -31,17 +31,17 @@ class QueryButtonsView extends Thorax.View
           , 3000
   initialize: ->
     @currentPopulation ?= 'IPP'
-  ipp: -> @model.ipp()
-  numerator: -> @model.numerator()
-  denominator: -> @model.denominator()
+  ipp: -> PopHealth.Helpers.formatNumber(@model.ipp())
```

Comment:
moved view methods to context as attributes, and used handlebars numeral to format